cmd/deploy/deploy.go

```diff
@@ -606,3 +590,35 @@ func (dc *DeployCommand) recreateFailedPods(ctx context.Context, name string) er
 	}
 	return nil
 }
+
+func (dc *DeployCommand) trackDeploy(manifest *model.Manifest, runInRemoteFlag bool, startTime time.Time, err error) {
+	deployType := "custom"
+	hasDependencySection := false
+	hasBuildSection := false
+	isRunningOnRemoteDeployer := false
+	if manifest != nil {
+		if manifest.IsV2 &&
+			manifest.Deploy != nil {
+			isRunningOnRemoteDeployer = isRemoteDeployer(runInRemoteFlag, manifest.Deploy.Image)
+			if manifest.Deploy.ComposeSection != nil &&
+				manifest.Deploy.ComposeSection.ComposesInfo != nil {
+				deployType = "compose"
+			}
+		}
+
+		hasDependencySection = manifest.HasDependencies()
+		hasBuildSection = manifest.HasBuildSection()
+	}
+
+	dc.AnalyticsTracker.TrackDeploy(analytics.DeployMetadata{
```

Comment:
I am getting a panic here when I try to deploy the movies repo:

```
Executing command 'Deploy Kafka'...
Release "kafka" does not exist. Installing it now.
Error: 1 error occurred:
	* Internal error occurred: failed calling webhook "serviceaccount.webhook.okteto.com": failed to call webhook: the server could not find the requested resource
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x18 pc=0x10682348c]

goroutine 55 [running]:
github.com/okteto/okteto/cmd/deploy.(*DeployCommand).trackDeploy(0x1400184e780, 0x1400184e840?, 0x1?, {0x14001724ea0?, 0x0?, 0x10949dcc0?}, {0x107a91280, 0x140006307a0?})
	github.com/okteto/okteto/cmd/deploy/deploy.go:613 +0x27c
github.com/okteto/okteto/cmd/deploy.Deploy.func1.1()
	github.com/okteto/okteto/cmd/deploy/deploy.go:217 +0x8c
created by github.com/okteto/okteto/cmd/deploy.Deploy.func1
	github.com/okteto/okteto/cmd/deploy/deploy.go:214 +0x738
```

Reply:
Fixed. Somehow I removed on a previous commit in the declaration 🤷 . Thanks for spotting it